metric-anomaly-data-model/src/main/java/org/hypertrace/alert/engine/metric/anomaly/datamodel/queue/KafkaAlertTaskConsumer.java

```diff
@@ -61,12 +60,7 @@ public void close() {
     baseProperties.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true");
     baseProperties.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, "1000");
     baseProperties.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "30000");
-    baseProperties.put(
-        ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG,
-        "org.apache.kafka.common.serialization.StringDeserializer");
-    baseProperties.put(
-        ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
-        "org.apache.kafka.common.serialization.ByteBufferDeserializer");
+    baseProperties.putAll(kafkaConfigReader.getConsumerSerdeProperties());
```

Comment:
Just a note, when you work on deployment, for this consumer component, we will need a higher value for `heartbeat.interval.ms` and `session.timeout.ms`.